<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailsWindow</class>
 <widget class="QDialog" name="DetailsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_bill_number">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Rachunek nr</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>420</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Koszt całkowity</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_costs">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>321</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_discard">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Anuluj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_ok">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>DRUKUJ RACHUNEK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_value">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>420</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>0,00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>450</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Koszt ze zniżką</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDiscount">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>450</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0,00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
